README.md

```diff
@@ -82,11 +82,13 @@ vars:
 Your Jira connector may not sync every table that this package expects. If you do not have the `SPRINT`, `COMPONENT`, or `VERSION` tables synced, add the respective variables to your root `dbt_project.yml` file. Additionally, if you want to remove comment aggregations from your `jira__issue_enhanced` model,  add the `jira_include_comments` variable to your root `dbt_project.yml`:
 ```yml
 vars:
-    jira_using_sprints: false   # Disable if you do not have the sprint table or do not want sprint-related metrics reported
-    jira_using_components: false # Disable if you do not have the component table or do not want component-related metrics reported
-    jira_using_versions: false # Disable if you do not have the versions table or do not want versions-related metrics reported
-    jira_using_priorities: false # disable if you are not using priorities in Jira
-    jira_include_comments: false # This package aggregates issue comments so that you have a single view of all your comments in the jira__issue_enhanced table. This can cause limit errors if you have a large dataset. Disable to remove this functionality.
+    jira_using_sprints: false    # Enabled by default. Disable if you do not have the sprint table or do not want sprint-related metrics reported.
+    jira_using_components: false # Enabled by default. Disable if you do not have the component table or do not want component-related metrics reported.
+    jira_using_versions: false   # Enabled by default. Disable if you do not have the versions table or do not want versions-related metrics reported.
+    jira_using_priorities: false # Enabled by default. Disable if you are not using priorities in Jira.
+    jira_include_comments: false # Enabled by default. Disabling will remove the aggregation of comments via the `count_comments` and `conversations` columns in the `jira__issue_enhanced` table.
+    jira_include_conversations: false # Disabled by default for Redshift, enabled by default for other supported warehouses. Controls only the `conversation` column in the `jira__issue_enhanced` table.  Disabling removes `conversation` columns but keeps the `count_comments` field if `jira_include_comments` is still enabled, which can help avoid limit errors with large datasets.
```

Comment:
I kinda wonder if this should be in its own section (or subsection of this section) , since there is no `conversation` source table.  It's a little different from the other variables here, so I fear this format might be a lil confusing. 

Also, if we do this, we can move the info about the default value out from the yml comment, which you currently need to scroll quite a bit to the right to see the end of https://github.com/fivetran/dbt_jira/tree/bug/string-agg-limit?tab=readme-ov-file#step-4-disable-models-for-non-existent-sources

@fivetran-catfritz

Reply:
Good point. I moved it!